### PR TITLE
Ft apollo server express

### PR DESCRIPTION
**Pull request by chrismatgit**
Please review my work... @verdotte

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
```
Comment (verdotte) [inline package-lock.json]: Don't push `package-lock.json` on GitHub, ignore it in `.gitignore` file

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (verdotte) [inline .babelrc:1]: Avoid having multiple commit in single branch, always use git commit --amend in order to update the current commit

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  resolvers,
+  context: {
+    models,
+    me: models.users[1],
+  },
+});
+
+server.applyMiddleware({ app, path: "/graphql" });
+
+app.listen({ port: 8000 }, () => {
```
Comment (verdotte) [inline src/index.js]: Don't hard code the value port value, kindly call it from the ENV file

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+const server = new ApolloServer({
+  typeDefs: schema,
+  resolvers,
+  context: {
+    models,
+    me: models.users[1],
+  },
+});
+
+server.applyMiddleware({ app, path: "/graphql" });
```
Comment (verdotte) [inline src/index.js]: Add the path value in the ENV file and call here from the ENV.

Comment (chrismatgit) [reply src/index.js]: Whenever i call it from the `.env`, it brings an error `Cannot POST /graphql`. My `.env` file looks like this:
`PATH='/graphql'
PORT=8000`. Kindly help me here... @verdotte 
Note that the port path is working but the only problem is on `process.env.PATH`

Comment (verdotte) [reply src/index.js]: u can declare it like that inside the ENV file **`GRAPHQL_URL=/graphql`**
and u call it that way **`const { GRAPHQL_URL } = process.env;`**

**Pull request merged**